Lib9c/Action/AccountStateDeltaExtensions.cs

```diff
@@ -148,6 +148,17 @@ public static AvatarState GetAvatarState(this IAccountStateDelta states, Address
             }
         }
 
+        public static bool TryGetAvatarState(
+            this IAccountStateDelta states,
+            Address agentAddress,
+            Address avatarAddress,
+            out AvatarState avatarState
+        )
+        {
+            avatarState = states.GetAvatarState(avatarAddress);
+            return !(avatarState is null) && avatarState.agentAddress == agentAddress;
```

Comment:
Can we do this address checking(`avatarState.agentAddress == agentAddress`) *before* full deserization by `GetAvatarState()`? it will be better if so.

Reply:
ok, i'll try